includes/class-reader-activation.php

```diff
@@ -669,6 +654,21 @@ public static function render_auth_form( $is_inline = false ) {
 							</p>
 						<?php endif; ?>
 					</form>
+					<div class="<?php echo \esc_attr( $class( 'response' ) ); ?>">
+						<span class="<?php echo \esc_attr( $class( 'response', 'icon' ) ); ?>" data-form-status="400">
+							<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="-2 -2 24 24" role="img" aria-hidden="true" focusable="false">
+								<path d="M10 2c4.42 0 8 3.58 8 8s-3.58 8-8 8-8-3.58-8-8 3.58-8 8-8zm1.13 9.38l.35-6.46H8.52l.35 6.46h2.26zm-.09 3.36c.24-.23.37-.55.37-.96 0-.42-.12-.74-.36-.97s-.59-.35-1.06-.35-.82.12-1.07.35-.37.55-.37.97c0 .41.13.73.38.96.26.23.61.34 1.06.34s.8-.11 1.05-.34z" />
+							</svg>
+						</span>
+						<span class="<?php echo \esc_attr( $class( 'response', 'icon' ) ); ?>" data-form-status="200">
+							<?php echo self::get_account_icon(); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
+						</span>
+						<div class="<?php echo \esc_attr( $class( 'response', 'content' ) ); ?>">
+							<?php if ( ! empty( $message ) ) : ?>
+								<p><?php echo \esc_html( $message ); ?></p>
+							<?php endif; ?>
+						</div>
+					</div>
```

Comment:
Repositioning the message caused CSS issues with the alert icon:

<img width="837" alt="Captura de Tela 2022-08-08 às 15 36 02" src="https://user-images.githubusercontent.com/820752/183489214-a32ce57a-e093-4db9-a461-0832963c5e6e.png">

Also, I wonder if we really need to change its position. The previous `form.replaceWith()` strategy gives us more design flexibility to place the message where it suits best.

Reply:
Ah, the reason I removed the `replaceWith` treatment is that `replaceWith` causes the form inputs to be removed from the DOM, which results in JS errors in the `handleAccountLinkClick` handler due to us using a different closure for this now. Maybe this is a step too far, though. I'll revisit.

Reply:
3f53a61 restores the `replaceWith` treatment and seems to avoid the JS errors I was experiencing before when preauthenticating.